client/scss/grids/_grid_scroll.scss

```diff
@@ -35,8 +35,22 @@
     }
 
     @include respond-to('xlarge') {
-      // (100vw - #{$container-width-max}) here accounts for any whitespace wide of the container max-width
-      min-width: calc(((100vw + #{map-get($gutter-width, 'large')} - (2 * #{map-get($container-padding, 'large')})) - (100vw - #{$container-width-max})) / 3);
+      min-width: ($container-width-max - (2 * map-get($container-padding, 'large')) + map-get($gutter-width, 'large')) / 3;
+      max-width: ($container-width-max - (2 * map-get($container-padding, 'large')) + map-get($gutter-width, 'large')) / 3;
+    }
+
+  }
+
+  &.grid--strip {
```

Comment:
This adds more items to the visible area of a `grid--scroll` element – wondering if it should be called something like `grid--scroll-6xl` instead? Can't decide whether that's clearer or not.